assets/css/custom.scss

```
@@ -3908,3 +3908,1439 @@ font-size: 1.55rem;
+        &--client img { height: 26px; }
+        &--mp img     { height: 22px; }
+      }
+    }
+
+    &__lockup-x {
+      font-family: $header-font;
+      font-size: 1.8rem;
+      font-weight: 300;
+      line-height: 1;
+      @include cs-brand-gradient-text;
+      user-select: none;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)